www/front_src/src/helpers/dynamicImport.js

```diff
@@ -64,9 +62,9 @@ export const dynamicImport = (basename, parameters) =>
 
     try {
       // dynamically import css if external component needs one
```

Comment:
We can get rid of this now

Reply:
We will get rid of it in the Module Federation PR. It's not related to that PR